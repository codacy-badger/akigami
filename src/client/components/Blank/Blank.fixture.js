import Blank from './Blank';

export default {
  component: Blank,
  props: {
    children: 'lorem ipsum',
  },
};
