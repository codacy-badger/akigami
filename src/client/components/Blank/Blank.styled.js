import styled from 'emotion/react';

export const Block = styled('div')`
    background-color: #f6f5f3;
    padding: 16px;
    text-align: center;
`;

export default Block;
