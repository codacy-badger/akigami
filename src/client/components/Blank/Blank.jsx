import React, { PureComponent } from 'react';
import PropTypes from 'prop-types';
import { Block } from './Blank.styled';

class Blank extends PureComponent {
    static propTypes = {
        children: PropTypes.any.isRequired,
    }
    render() {
        const { children } = this.props;
        return (
            <Block>
                {children}
            </Block>
        );
    }
}

export default Blank;
