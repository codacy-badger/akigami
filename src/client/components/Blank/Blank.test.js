import { create as renderer } from 'react-test-renderer';
import createTestContext from 'react-cosmos-test/generic';
import Blank from './Blank.fixture';

const res = createTestContext({ renderer, fixture: Blank });
beforeEach(res.mount);
test('Blank snapshot', () => {
  expect(res.getWrapper().toJSON()).toMatchSnapshot();
});

