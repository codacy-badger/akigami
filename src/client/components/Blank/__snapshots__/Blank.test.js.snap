// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Blank snapshot 1`] = `
<div
  className="css-1tgf6q4"
>
  lorem ipsum
</div>
`;
